src/client/swagger/unity.yml

```diff
@@ -727,7 +727,7 @@ components:
         balanceThreshold:
           description: threshold for notification
           type: string
-        notify:
+        isNotify:
```

Comment:
nice catch. We should be consistent about field names used in different places.

Reply:
So in going over this with Subir earlier today I wrote this up:

https://github.com/influxdata/ui/issues/946

@ischolten I'm not sure what the difference is between the two, or what the intent is. I know that the two are distinct parts of the billing and checkout page, but it looks like the two both have the same properties and I'm not sure if there's a reason we have the two split between different API contracts and types

Reply:
Right now, the checkout api functions as kind of a transactional api. So its creating a billing threshold, updating a billing contact, saving a payment id and upgrading the account and org limits. vs when youre on the billing page, we want to be able to see just the billing threshold and update just the billing threshold. Which is why these are separate.

Reply:
Cool, so for now it seems like the two can stay separate until the swagger contract moves over to Quartz, at which point we can probably just create embedded types for the notification types. I'm gonna close out that issue